src/components/admin/StaffListController.jsx

```diff
@@ -90,8 +89,7 @@ class StaffListController extends FalcorController {
 
     // Check if the slug is already taken
     this.props.model.get(['staff', 'bySlug', slug, 'slug']).then(x => {
-      if (x) {
-        x = cleanupFalcorKeys(x); // eslint-disable-line no-param-reassign
+      if (x.json.staff.bySlug[slug].slug !== undefined) {
```

Comment:
This will fail if `x == undefined` no? Use `_.has`.

Reply:
No it won't! :D. Because this is me utilizing the new falcor update that consistently returns the whole object structure you requested :)